.github/workflows/fmtcheck.yml

```diff
@@ -22,6 +22,7 @@ jobs:
            github.com:443
            api.github.com:443
            proxy.github.com:443
+           proxy.golang.org:443
```

Comment:
why is this needed? I can't recall when the proxy was introduced but I seem to recall it was prior to 1.20

Reply:
I have no major objections here so I guess we can move on